crates/energy/src/battery/mod.rs

```diff
@@ -0,0 +1,56 @@
+use bevy::prelude::*;
+
+pub(crate) mod component;
+mod systems;
+
+pub(crate) struct BatteryPlugin;
+
+impl Plugin for BatteryPlugin {
+    fn build(&self, app: &mut App) {
+        app.add_system(systems::discharge_battery);
+    }
+}
+
+/// The rate at which the battery discharges in kiloJoules per second.
```

Comment:
I would use base units from SI (we can convert in UI) unless there is a technical reason not to do so (e.g. fixed point arthritics).